config/configmap/cro-configmaps.yaml

```diff
@@ -10,4 +10,6 @@ data:
   self-managed: >
     {"blobstorage":"openshift", "smtpcredentials":"openshift", "redis":"openshift", "postgres":"openshift"}
   managed-api: >
+    {"blobstorage":"openshift", "smtpcredentials":"openshift", "redis":"openshift", "postgres":"openshift"}
+  multitenant-managed-api: >
```

Comment:
Not something for this pr or jira but at some stage we may want to update the sample config map in cro
https://github.com/integr8ly/cloud-resource-operator/blob/master/config/samples/cloud_resource_config.yaml

Reply:
I think we want to keep the cro one integreatly agnostic so this change shouldn't impact the config map in cro.